mmedit/models/extractors/feedback_hour_glass.py

```diff
@@ -0,0 +1,90 @@
+import torch.nn as nn
+
+
+class ResBlock(nn.Module):
+    """ResBlock for HourGlass.
+
+    It has a style of:
+
+    ::
+
+        ---Conv-ReLU-Conv-Conv-+-
+         |_________Conv________|
+
+        or
+
+        ---Conv-ReLU-Conv-Conv-+-
+         |_____________________|
+
+    Args:
+        in_channels (int): Number of channels in the input features.
+        out_channels (int): Number of channels in the output features.
+    """
+
+    def __init__(self, in_channels, out_channels):
+        super().__init__()
+        self.conv_block = nn.Sequential(
+            nn.Conv2d(in_channels, out_channels // 2, 1),
+            nn.ReLU(inplace=True),
+            nn.Conv2d(
+                out_channels // 2, out_channels // 2, 3, stride=1, padding=1),
+            nn.Conv2d(out_channels // 2, out_channels, 1))
+        if in_channels == out_channels:
+            self.skip_layer = None
+        else:
+            self.skip_layer = nn.Conv2d(in_channels, out_channels, 1)
+
+    def forward(self, x):
+        """Forward function.
+
+        Args:
+            x (Tensor): Input tensor with shape (n, c, h, w).
+
+        Returns:
+            Tensor: Forward results.
+        """
+
+        residual = self.conv_block(x)
+        if self.skip_layer:
+            x = self.skip_layer(x)
+        return x + residual
+
+
+class HourGlass(nn.Module):
```

Comment:
Any relationship with this? https://github.com/open-mmlab/mmpose/blob/master/mmpose/models/backbones/hourglass.py

Reply:
Refer to this file.